stable/database/templates/persistentvolumeclaim.yaml

```diff
@@ -37,4 +37,33 @@ spec:
       storage: {{ .Values.database.sm.hotCopy.persistence.size | quote }}
   storageClassName: {{ .Values.database.sm.hotCopy.persistence.storageClass }}
 {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+{{- if .Values.database.te.logPersistence.enabled }}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.database.name }}-log-te-volume
+  labels:
+    app: {{ template "database.fullname" . }}
+    group: nuodb
+    database: {{ .Values.database.name }}
+    domain: {{ .Values.admin.domain }}
+    chart: {{ template "database.chart" . }}
+    release: {{ .Release.Name | quote }}
+spec:
+  accessModes:
+  {{- range .Values.database.te.logPersistence.accessModes }}
+    - {{ . }}
+  {{- end }}
+  {{- if .Values.database.te.logPersistence.storageClass }}
+  {{- if (eq "-" .Values.database.te.logPersistence.storageClass) }}
```

Comment:
Do we need to allow anything other than RWM?

Reply:
I did it this way to make it more similar to SM/Admins. Put the value in the values file, rather than hide it here.

Of course, as far as we know, we can only accept RWM.